rollup.config.js

```diff
@@ -1,3 +1,9 @@
+/* eslint-disable import/no-commonjs */
+/* eslint-env es6 */
+
+
+const babel = require('rollup-plugin-babel');
```

Comment:
this worked fine without babel previously, so I don't think we'd have to add it

Reply:
👍